boards/arc/nsim/doc/index.rst

```diff
@@ -7,20 +7,23 @@ Overview
 ********
 
 This board configuration will use `Designware ARC nSIM`_ to emulate a virtual
-ARC EM based board including the following features:
+ARC EM or ARC HS based board including the following features:
 
-* ARC EM processor
+* ARC EM or ARC HS processor
 * ARC internal timer
 * a virtual output only console (uart-nsim)
 
-There are two sub configurations in board:
+There are three supported board sub-configurations:
 
-* nsim_em which includes normal em features and ARC MPUv2
-* nsim_sem which includes secure em features and ARC MPUv3
+* ``nsim_em`` which includes normal ARC EM features and ARC MPUv2
+* ``nsim_sem`` which includes secure ARC EM features and ARC MPUv3
+* ``nsim_hs`` which includes base ARC HS features, i.e. w/o PMU and MMU
 
 For detailed arc features, please refer to
 :zephyr_file:`boards/arc/nsim/support/nsim_em.props` and
 :zephyr_file:`boards/arc/nsim/support/nsim_sem.props`.
+:zephyr_file:`boards/arc/nsim/support/nsim_hs.props`
+>>>>>>> board/nsim: Add support of ARC HS cores in nSIM
```

Comment:
Is this line intended? Looks like a diff leftover ?

Reply:
This is a part of my not yet merged pull-request https://github.com/zephyrproject-rtos/zephyr/pull/16662 and I spotted that thing yesterday and already pushed a fixed version.